tests/test_pdf_chunks.py

```diff
@@ -0,0 +1,71 @@
+import os
+import pytest
+import pixeltable as pxt
+from typing import List
+from pixeltable.iterators.pdf_page_extractor import PdfPageExtractor  # adjust path as needed
+from pixeltable.catalog import Catalog
+
+class TestPdfExtraction:
```

Comment:
What is this test doing that's different from the other one? Would just one of them suffice and/or can they be condsolidated into one test class?